engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/new/content-view-new.controller.js

```diff
@@ -46,5 +46,10 @@ angular.module('Bastion.content-views').controller('NewContentViewController',
             }
         });
 
+        $scope.$watch('contentView.composite', function () {
+            if (!$scope.contentView.composite) {
+                $scope.contentView.auto_publish = false; // eslint-disable-line camelcase
```

Comment:
eslint doesn't seem to know or care about this disable comment @johnpmitsch do you know why?

Reply:
My guess is because bastion uses a [very old version of eslint](https://github.com/Katello/bastion/blob/6e1ddbe30365963335e4675c61be765e6376652b/package.json#L5) that may not recognize this syntax

Try something [like this](https://eslint.org/docs/1.0.0/user-guide/configuring#configuring-rules) which I think is being used elsewhere [here](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/tasks/task.factory.js#L99)